src/Lambda/Result/AddLayerVersionPermissionResponse.php

```diff
@@ -34,7 +34,7 @@ public function getStatement(): ?string
 
     protected function populateResult(ResponseInterface $response, HttpClientInterface $httpClient): void
     {
-        $data = json_decode($response->getContent(false), true);
+        $data = $response->getContent(false)->toArray();
```

Comment:
you means `$response->toArray(false)` right?

Reply:
I suppose, depending on how you want to deal with 300-599 here :)